torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3897,6 +3912,16 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            supports_inplace_autograd=True,
            safe_casts_outputs=True,
            sample_inputs_func=sample_inputs_xlogy),
+    OpInfo('logsumexp',
+           dtypes=floating_types_and(torch.bfloat16),
+           supports_out=False,
+           supports_complex_autograd=False,
+           skips=(
+               # logsumexp get error for the following test.
```

Comment:
can you elaborate on this error?

Reply:
yes and the error message is as following for the given test : 

Failure in testing nodes' autodifferentiation. One or more nodes were not expected to be autodiffed but were found in a DifferentiableGraph or in a FusionGroup of a DifferentiableGraph. Did you intend for these nodes to be autodiffed? If so, change this test to expect autodifferentiation. 
Specifically:
  ['aten::logsumexp'] were not expected to be in one of the DifferentiableGraphs but were.

Reply:
I see. You'll need to play around with the autodiff metadata, see here:

https://github.com/pytorch/pytorch/blob/facbcec2980783a011a046643577592480754210/torch/testing/_internal/common_methods_invocations.py#L150

Based on the error message, I think the change this needs is

```
assert_autodiffed=True
```